src/uwtools/tests/drivers/test_driver.py

```diff
@@ -272,8 +279,9 @@ def test_Driver__validate(driverobj):
 
 def test_Driver__write_runscript(driverobj, tmp_path):
     path = tmp_path / "runscript"
+    executable = driverobj._driver_config["execution"]["executable"]
     driverobj._write_runscript(path=path, envvars={"FOO": "bar", "BAZ": "qux"})
-    expected = """
+    expected = f"""
```

Comment:
Is that three double quotes as an f-string? What is happening?

Reply:
Did you not know this was a thing? I'm a little confused.

Reply:
Oh good grief, how embarrassing: I was looking at something else, got confused, and didn't notice that this was the start of a multiline f-string...

Reply:
That sounds like something I'd do. I was right to be confused.